packages/ui/src/components/Navigation/Navigation.stories.tsx

```diff
@@ -0,0 +1,880 @@
+                    slug: 'games',
+                },
+            },
+        },
+        {
+            id: 6603,
+            display_name: 'Animals',
+            display_description: '',
+            stories_number: 1,
+            public_stories_number: 1,
+            i18n: {
+                en: {
+                    locale: {
+                        code: 'en',
+                        locale: 'en',
+                        name: 'English (Global)',
+                        native_name: 'English (Global)',
+                        direction: 'ltr',
+                        language_code: 'en',
+                    },
+                    name: 'Animals',
+                    description: '',
+                    slug: 'animals',
+                },
+            },
+        },
+        {
+            id: 6604,
+            display_name: 'Sports',
+            display_description: '',
+            stories_number: 1,
+            public_stories_number: 1,
+            i18n: {
+                en: {
+                    locale: {
+                        code: 'en',
+                        locale: 'en',
+                        name: 'English (Global)',
+                        native_name: 'English (Global)',
+                        direction: 'ltr',
+                        language_code: 'en',
+                    },
+                    name: 'Sports',
+                    description: '',
+                    slug: 'sports',
+                },
+            },
+        },
+        {
+            id: 6605,
+            display_name: 'Other',
+            display_description: '',
+            stories_number: 1,
+            public_stories_number: 1,
+            i18n: {
+                en: {
+                    locale: {
+                        code: 'en',
+                        locale: 'en',
+                        name: 'English (Global)',
+                        native_name: 'English (Global)',
+                        direction: 'ltr',
+                        language_code: 'en',
+                    },
+                    name: 'Other',
+                    description: '',
+                    slug: 'other',
+                },
+            },
+        },
+        {
+            id: 6606,
+            display_name: 'Family',
+            display_description: '',
+            stories_number: 1,
+            public_stories_number: 1,
+            i18n: {
+                en: {
+                    locale: {
+                        code: 'en',
+                        locale: 'en',
+                        name: 'English (Global)',
+                        native_name: 'English (Global)',
+                        direction: 'ltr',
+                        language_code: 'en',
+                    },
+                    name: 'Family',
+                    description: '',
+                    slug: 'family',
+                },
+            },
+        },
+    ] as Category[],
+    languages: [
```

Comment:
Same here, these objects are too large and make stories unreadable. Please extract into a separate mock file.